guides/keras_cv/semantic_segmentation_deeplab_v3_plus.py

```diff
@@ -0,0 +1,322 @@
+"""
+Title: Semantic Segmentation with KerasCV
+Author: Divyashree Sreepathihalli, Ian Stenbit
```

Comment:
You might want to link to your Github profile or social media.

Reply:
done